cmd/lncli/commands.go

```diff
@@ -69,13 +69,24 @@ func printRespJSON(resp proto.Message) {
 func actionDecorator(f func(*cli.Context) error) func(*cli.Context) error {
 	return func(c *cli.Context) error {
 		if err := f(c); err != nil {
+			s, ok := status.FromError(err)
+
+			// If it's a command for the UnlockerService (like 'create' or
+			// 'unlock') but the wallet is already unlocked, then these methods
```

Comment:
lines should be wrapped at 80 chars.

Reply:
They were, actually. But because of the three levels of indentation it looks like more in the GitHub diff view.
I wrapped the longest line anyway.

Reply:
Ok, I think the default tab width in my IDE is wrong, sorry. I‘ll re-format it again.